mirrord/cli/src/verify_config.rs

```diff
@@ -182,13 +189,36 @@ pub(super) async fn verify_config(VerifyConfigArgs { ide, path }: VerifyConfigAr
         });
 
     let verified = match layer_config {
-        Ok(config) => VerifiedConfig::Success {
-            config: config.target.into(),
-            warnings: config_context.get_warnings().to_owned(),
-            compatible_target_types: TargetType::all()
-                .filter(|tt| tt.compatible_with(&config.feature))
-                .collect(),
-        },
+        Ok(config) => {
+            let verified_target_config = VerifiedTargetConfig::from(config.target);
+
+            // Check if we're dealing with an operator-only target, and `config.operator`
+            // was set to `false` (operator purposefully disabled).
+            if let Some((verified_target, false)) = verified_target_config
+                .path
+                .clone()
+                .map(VerifiedTarget::from)
+                .zip(config.operator)
+                && verified_target.requires_operator()
+            {
+                VerifiedConfig::Fail {
+                    errors: vec![
+                        "{verified_target} target requires the mirrord-operator, but the config \
+                        was set to `operator: false`! Consider setting `operator: true` for \
+                        this target type."
```

Comment:
```suggestion
                        format!("{} targets require the mirrord-operator, but operator usage was explicitly disabled. Consider enabling mirrord-operator in your mirrord config.", verified_target.target_type())
```
Requires implementing `VerifiedTarget::target_type(&self) -> TargetType` and `Display for TargetType` but would be nicer, I think

Reply:
Changed. I ended up impl'ing `Into`, wdyt?

Reply:
Noice :+1: